i18next/i18next.d.ts

```diff
@@ -87,6 +87,7 @@ interface I18nextStatic {
     };
     init(callback?: (t: (key: string, options?: any) => string) => void ): JQueryDeferred<any>;
     init(options?: I18nextOptions, callback?: (t: (key: string, options?: any) => string) => void ): JQueryDeferred<any>;
+    init(options?: I18nextOptions, callback?: (err: any, t: (key: string, options?: any) => string) => void ): JQueryDeferred<any>;

```

Comment:
Thank you for the comment. I removed the old lines and applied the same modification to the other parts with the same callback.